docs/release-notes/release-2022-04-11.md

```diff
@@ -0,0 +1,116 @@
+---
+title: April 2023
+---
+
+# Release Notes April 2023 (2023-04-11)
+
+:::danger
+This release contains **breaking changes**
+:::
+
+## New Features
+
+### @typespec/compiler
+
+- Emit diagnostic when ambiguous symbol exists between global and usings
+- Add support for referencing metatypes ModelProperty `::type` and Operation `::returnType` and `::parameters`
+- Relax constraint on derived type overrides
+- Prevent use of augment decorators on instantiated templates.
+- Allow projectedNames helpers to work with previous projections
+- Add helper `interpolatePath` for emitter to do additional interpolation on config properties
+- Api: Update `getTypeName` to omit standard library namespace
+- Allow overloads in interfaces to work under projection
+- Add pre-projection support
+- Provide access to extended interfaces in type graph
+- Document member ordering and fix projection rename reordering
+
+### @typespec/openapi3
+
+- Support shared routes
+- Update to use new `interpolatePath` logic to resolve the `output-file`
+
+### @typespec/migrate
+
+- Add migrate script for zonedDateTime
+- Add migration script for `@header` and `@query` format required
+
+## Bug Fixes
+
+### @typespec/compiler
+
+- Fix: emitter framework will now visit scalar declarations
+- Fix: Alias unknown ref crash
+- Fix: Empty model expression assignable to array
+- Fix: `tsp code uninstall` not finding extension to uninstall
+- Fix: Issue where template parameter type check wouldn't work if constraint is exact same type as next validation
+
+### @typespec/rest
+
+- Fix: Issue where `@action("")` was treated the same as `@action`. Now this emits an error.
+
+### @typespec/versioning
+
+- Fix: Issue with using version as a template parameter across different namespaces. Includes a significant change in the versioning library internals.
+- Fix:Use pre-projections to fix issues with versioned resources.
+
+### VS Code extension
+
+- Fix: Highlighting of TypeSpec code blocks in markdown.
+
+## Breaking Changes
+
+### @typespec/versioning: Removed deprecated versioning symbols.
+
+- Removed `@versionedDependency` decorator. Use `@useDependency` instead.
```

Comment:
Not for your PR but I think we have a lot of docs to update for this in typespec-azure